cmake/BLTMacros.cmake

```diff
@@ -217,17 +217,26 @@ macro(blt_add_target_link_flags)
          "${options}" "${singleValuedArgs}" "${multiValuedArgs}" ${ARGN} )
 
     if(NOT "${arg_FLAGS}" STREQUAL "")
-        # get prev flags
-        get_target_property(_LINK_FLAGS ${arg_TO} LINK_FLAGS)
-        if(NOT _LINK_FLAGS)
-            set(_LINK_FLAGS "")
+        if( ${CMAKE_VERSION} VERSION_GREATER_EQUAL "3.13.0" )
+            # In CMake 3.13+, LINK_FLAGS was converted to LINK_OPTIONS.
+            # This now supports generator expressions but expects a list
+            # not a string
+
+            # Convert from a string to a CMake ;-list
+            string (REPLACE " " ";" _link_flags_list "${arg_FLAGS}")
```

Comment:
This breaks any flags with spaces?

Reply:
Yes this was definitely going to cause problems.  Good catch.  I removed it because this macro required a list anyways and fixed the one place in BLT we were violating that (MPI).